.kitchen.yml

```diff
@@ -4,22 +4,20 @@ driver:
 
 provisioner:
   name: chef_zero
+  require_chef_omnibus: 13
 
 platforms:
   - name: ubuntu-16.04
   - name: ubuntu-14.04
-  - name: ubuntu-12.04
-  - name: debian-8.6
-  - name: debian-7.11
-  - name: centos-6.8
-  - name: centos-7.3
+  - name: debian-9.2
```

Comment:
You can change these to just debian-9 to get the slugs we push up for Bento. Every bento release updates them and as long as you keep the boxes up to date you'll always have the latest point release.